docs/reference/search/search-your-data/behavioral-analytics/behavioral-analytics-overview.asciidoc

```diff
@@ -1,4 +1,7 @@
 [[behavioral-analytics-overview]]
+
+deprecated:[9.0.0]
+
 == Search analytics
```

Comment:
```suggestion
[[behavioral-analytics-overview]]
== Search analytics

deprecated:[9.0.0]

```

Reply:
not sure if this is messing with doc links but let's see

Reply:
Maybe. Let's test it out